test/integration/gobierto_people/people/person_events_index_test.rb

```diff
@@ -109,33 +109,31 @@ def test_person_events_present_in_other_agendas
         nelson_event.attendees.create(person: richard)
 
         with(site: site, js: true) do
-          visit gobierto_people_person_events_path(richard.slug)
+          visit @path
+
+          click_button "List"
```

Comment:
I prefer to test it like the other tests and don't depend on the selector of the calendar